recipes/libpcap/all/conanfile.py

```diff
@@ -30,12 +30,14 @@ class LibPcapConan(ConanFile):
         "fPIC": [True, False],
         "enable_libusb": [True, False],
         "with_snf": [True, False],
+        "with_dpdk": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "enable_libusb": False,
         "with_snf": False,
+        "with_dpdk": False,
```

Comment:
You mention that without this option, the recipe does not build, which leads me to belive that this is True in upstream's scripts when nothing is said.

If this makes all builds fail, we could make an expcetion here and diverge from upstream, but the CCI policy is to follow Upstream choices as closely as possible. Any insight into the proper default value? Thanks!

Reply:
I'm sorry, I probably misled you, errors occur with the `--with-dpdk` flag, which is enabled by default in libpcap. But the problem appears only if dpdk lib is installed. To avoid errors, you need the `--without-dpdk` flag, which is set by `"with_dpdk": False`, which I set by default.